crates/flipperzero/src/furi/string.rs

```diff
@@ -295,63 +295,59 @@ impl FuriString {
         Bytes(self.to_bytes().iter().copied())
     }
 
-    /// Returns `true` if the given pattern matches a sub-slice of this string slice.
+    /// Returns `true` if the given pattern `pat` matches a sub-slice of this string slice.
     ///
     /// Returns `false` if it does not.
     ///
-    /// The pattern can be a `&FuriString`, [`c_char`], `&CStr`, [`char`], or a slice of
-    /// [`char`]s.
+    /// The pattern can be a `&FuriString`, [`c_char`], `&CStr`, [`char`],
+    /// or a slice of [`char`]s.
```

Comment:
Undo this change; the line wrap length was correct for this file (90 characters).
```suggestion
    /// The pattern can be a `&FuriString`, [`c_char`], `&CStr`, [`char`], or a slice of
    /// [`char`]s.
```

Reply:
Afaik, the [default](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#doc_comment_code_block_width) for docomment width is also `100` so we don't even need tro split the lines.

Although if you insist, we could enforce `90` though it does not seems reasonable to me (and, in this speific case, 100 looks better IMO).

Same goes for the other similar comments although you ma

Reply:
That default is for code snippets in doc comments, not for documentation itself. I presume it's set to 100 because that is the same break point as for regular code.

I use 90 characters as a break point for documentation because it hits a nice spot for IDE and terminal development readability. 100 for code works well because it's rare you have lines that long, and when the formatter does trigger a break the resulting lines end up much shorter. For documentation, that is not the case; the break point ends up forming a block boundary as there are more granular points to break at, so you end up with mostly full-width lines, and for larger paragraphs 100 is too wide for easy reading. The research done on website UX tends to show that readability drops off at line lengths above 80-85 characters, so including the 4-8 character prefixes on doc lines, 90 sits at a nice break point for readability.

But in any case, what I care about more is minimizing PR diffs. Every change consumes reviewer energy, while they determine if it is semantically meaningful or not. If less energy is spent verifying that a change is just altering formatting, more energy is available for noticing bugs. So I prefer formatting-only changes to be separated out into separate PRs or commits, and ideally automated (though sadly AFAIK `rustfmt` doesn't have even optional support for formatting documentation).

Reply:
A very good point! Thanks for the detailed explanation.

In this case, I will revert the changes to this lines.

Also, let's add this experimental option to the formatting file in the future so that it is more explicit for other committers.